Tools/terrain-tools/terrain_rangefinder_postprocess.ipynb

```diff
@@ -254,7 +254,7 @@
     "    if col in cols:\n",
     "        print(f\"Adding {col}\")\n",
     "        y = pd.to_numeric(df[col], errors='coerce')\n",
-    "        \n",
+    "\n",
```

Comment:
Change in the tool's output, which may or may not be significant.  Has this been tested?